src/Microsoft.DotNet.Interactive.Http.Tests/ParserTests.Variables.cs

```diff
@@ -283,5 +283,35 @@ public void spaces_after_variable_do_not_produce_diagnostics()
 
             result.SyntaxTree.RootNode.ChildNodes.Count().Should().Be(1);
         }
+
+        [Fact]
+        public void single_quotes_in_variable_values_are_supported()
+        {
+            var result = Parse(
+                """
+
+                @host='https://httpbin.org'
+                """
+                );
+
+            var variables = result.SyntaxTree.RootNode.TryGetDeclaredVariables().declaredVariables;
+            variables.Should().Contain(n => n.Key == "host").Which.Value.Should().BeOfType<DeclaredVariable>().Which.Value.Should().Be("'https://httpbin.org'");
+
+        }
+
+        [Fact]
+        public void double_quotes_in_variable_values_are_supported()
+        {
+            var result = Parse(
+                """
+
+                @host="https://httpbin.org"
+                """
+                );
+
+            var variables = result.SyntaxTree.RootNode.TryGetDeclaredVariables().declaredVariables;
+            variables.Should().Contain(n => n.Key == "host").Which.Value.Should().BeOfType<DeclaredVariable>().Which.Value.Should().Be("\"https://httpbin.org\"");
```

Comment:
This seems a little surprising to me that the quotes are part of the variable value.

Maybe related, are spaces in variable values supported like this then?

```
@blah = one two three
```